server/controllers/auth.js

```diff
@@ -84,12 +84,48 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
   }
 });
 
+// @route POST /auth/demo-login
+// @desc Login demo user
+// @access Public
+exports.demoLogin = asyncHandler(async (req, res, next) => {
+
+  const { email, password } = req.body;
+
+  if (!email || !password || email !=="demo@kanban.com" || password !== "123456") {
+    res.sendStatus(400)
+  }
```

Comment:
I see you have actually created a route in the back-end, good work here. But the implementation should be different. We just want the route to not accept a body and the credentials are solely on the back-end in environment vars for email/password. You could even write up a script that other devs could run to generate their demo account based on the env vars. 👍🏻